modules/extensions-support/src/main/java/org/mule/module/extension/internal/introspection/AnnotationsBasedDescriber.java

```diff
@@ -343,6 +354,25 @@ private void declareOperationParameters(Method method, OperationDescriptor opera
             {
                 operation.withModelProperty(ConnectionTypeModelProperty.KEY, new ConnectionTypeModelProperty(parsedParameter.getType().getRawType()));
             }
+
+            UseConfig useConfig = parsedParameter.getAnnotation(UseConfig.class);
+            if (useConfig != null)
+            {
+                operation.withModelProperty(UseConfigTypeModelProperty.KEY, new UseConfigTypeModelProperty(parsedParameter.getType().getRawType()));
+            }
+        }
+    }
+
+    private void checkAnnotationIsNotUsedMoreThanOnce(Method method, Class annotationClass)
+    {
+        Stream<java.lang.reflect.Parameter> parametersStream = Arrays
+                .asList(method.getParameters())
+                .stream()
+                .filter(parameter -> parameter.isAnnotationPresent(annotationClass));
+
+        if (parametersStream.collect(Collectors.toList()).size() > 1)
```

Comment:
don't collect into the if clause... collect to a list and use the list moving forward